config/build.config.js

```diff
@@ -3,7 +3,7 @@ var fs = require('fs');
 var versionFile = __dirname + '/../dist/commit';
 
 module.exports = {
-  ngVersion: '1.4.3',
+  ngVersion: '1.3.15',
```

Comment:
This will not be merged; it is, however, useful for testing the Karma failures with Angular 1.3.x